bench/micro/Bench/Database/LSMTree/Internal/Merge.hs

```diff
@@ -362,9 +363,15 @@ createRun ::
   -> Run.RunFsPaths
   -> [SerialisedKOp]
   -> IO (Run IO (FS.Handle h))
-createRun hasFS hasBlockIO mMappend targetPath =
-      Run.fromWriteBuffer hasFS  hasBlockIO Run.CacheRunData (RunAllocFixed 10) targetPath
-    . Fold.foldl insert WB.empty
+createRun hasFS hasBlockIO mMappend targetPath kops = do
+    let blobpath = FS.addExtension (runBlobPath targetPath) ".wb"
+    wbblobs <- WBB.new hasFS blobpath
+    kops' <- traverse (traverse (traverse (WBB.addBlob hasFS wbblobs))) kops
+    let wb = Fold.foldl insert WB.empty kops'
+    run <- Run.fromWriteBuffer hasFS hasBlockIO Run.CacheRunData (RunAllocFixed 10)
+                               targetPath wb wbblobs
+    WBB.removeReference wbblobs
+    return run
```

Comment:
This looks a bit like `mkRunFromSerialisedKOps`. Maybe `mkRunFromSerialisedKOps` should be added to the `extras` library

Reply:
You did part of this in #407. I'll try and reuse that...

Reply:
Done.